context-free.md

```diff
@@ -2,21 +2,27 @@
 
 ##  Who is this programming language for?
 
+This programming language appears to be for anyone interested in vector-based digital art that does not want to go through the hassle of learning a more complete graphics editor, like Photoshop. Although the language itself is not extremely complicated, it seems geared toward someone who has some programming experience.
 
 ## What is easy to do in this language? Why is it easy?
 
+I found it simple to generate recurring simple shapes because most of the basic shapes are built in (SQUARE, TRIANGLE, CIRCLE) and the language seems to lend itself to recursion quite nicely.
 
 ## What is hard to do in this language? Why is it hard?
 
```

Comment:
You and I approached this question differently--I answered it more from the stand point of implementation whereas you considered the output. It's interesting to think about difficulties from both sides of the spectrum (coding and results). So for mine, I focused on the logistics of placement, which I found difficult.